src/stores/useToolStore/index.ts

```diff
@@ -0,0 +1,86 @@
+import { client } from '@/api/client';
+import { components } from '@/api/schema';
+import { OsaaminenValue } from '@/components';
+import { EhdotusData, ehdotusDataToRecord, EhdotusRecord } from '@/routes/Tool/utils';
+import { create } from 'zustand';
+// import { createJSONStorage, persist } from 'zustand/middleware';
```

Comment:
Is this comment left on purpose for future or why it is here? If it is left on purpose, it could have some descriptive comment to tell that.